internal/driver/fetch.go

```diff
@@ -429,10 +431,12 @@ mapping:
 			if m.File != "" {
 				// Try both the basename and the full path, to support the same directory
 				// structure as the perf symfs option.
-				if baseName != "" {
-					fileNames = append(fileNames, filepath.Join(path, baseName))
-				}
+				fileNames = append(fileNames, filepath.Join(path, baseName))
 				fileNames = append(fileNames, filepath.Join(path, m.File))
+				// Try other common locations
```

Comment:
"other common locations" is not very specific. Could you copy some words from the PR description to here to make this a bit cleerer? https://sourceware.org/gdb/onlinedocs/gdb/Separate-Debug-Files.html is the most important bit.

Reply:
I have extended the comment (included the link), and added the Yocto example to the commit message.